src/colornames.csv

```diff
@@ -53,6 +53,7 @@ Absinthe Turquoise,#008a60,
 Absolute Apricot,#ff9944,
 Absolute Zero,#0048ba,
 Abstract,#e4cb97,
+Abstract Idea,#bdd2de,

```

Comment:
Lets remove that one. I don't think it conveys a clear color. What do you think?